test/elm/interval/data.coffee

```diff
@@ -104369,8 +105845,172 @@ define HourPrecPerSecond: expand { Interval[@T01+00:00, @T01+00:00] } per second
 define HourPrecPerMillisecond: expand { Interval[@T01+00:00, @T01+00:00] } per millisecond
 ###
 
+###
+Translation Error(s):
+[4:41, 4:60] Could not resolve call to operator Add with signature (System.Time,System.Ratio).
+[5:43, 5:62] Could not resolve call to operator Add with signature (System.Time,System.Ratio).
+[6:43, 6:62] Could not resolve call to operator Add with signature (System.Time,System.Ratio).
+[7:48, 7:67] Could not resolve call to operator Add with signature (System.Time,System.Ratio).
+[9:42, 9:57] Could not resolve call to operator Add with signature (System.Time,System.Ratio).
+[10:44, 10:59] Could not resolve call to operator Add with signature (System.Time,System.Ratio).
+[11:44, 11:59] Could not resolve call to operator Add with signature (System.Time,System.Ratio).
+[12:49, 12:64] Could not resolve call to operator Add with signature (System.Time,System.Ratio).
+[14:42, 14:54] Could not resolve call to operator Add with signature (System.Time,System.Ratio).
+[15:44, 15:56] Could not resolve call to operator Add with signature (System.Time,System.Ratio).
+[16:44, 16:56] Could not resolve call to operator Add with signature (System.Time,System.Ratio).
+[17:49, 17:61] Could not resolve call to operator Add with signature (System.Time,System.Ratio).
+[19:43, 19:52] Could not resolve call to operator Add with signature (System.Time,System.Ratio).
+[20:45, 20:54] Could not resolve call to operator Add with signature (System.Time,System.Ratio).
+[21:45, 21:54] Could not resolve call to operator Add with signature (System.Time,System.Ratio).
+[22:50, 22:59] Could not resolve call to operator Add with signature (System.Time,System.Ratio).
```

Comment:
We have a task in the backlog to look into these errors and the ones below, but we have prioritized that investigation as lower than the standard updates.  I believe that @jbradl11 asked bryn about them and it was determined that these errors are likely things we need to update rather than issues with cql-to-elm.